library/item/item_setting/src/main/java/org/quiltmc/qsl/item/setting/api/QuiltCustomItemSettings.java

```diff
@@ -22,8 +22,7 @@
  * A list of the {@link CustomItemSetting}s that are provided by Quilt
  */
 public final class QuiltCustomItemSettings {
-	private QuiltCustomItemSettings() {
-	}
+	private QuiltCustomItemSettings() {}
```

Comment:
These contracts arent all correct. Check what happens for null in some of these.